packages/@angular/cli/blueprints/class/index.ts

```diff
@@ -11,11 +13,18 @@ export default Blueprint.extend({
       name: 'spec',
       type: Boolean,
       description: 'Specifies if a spec file is generated.'
+    },
+    {
+      name: 'app',
```

Comment:
If we're doing that we should also be doing the same thing with things like `flat` and `spec` too